data/plugins/clickhouse.ts

```diff
@@ -0,0 +1,90 @@
+import { Plugin } from "./index";
+
+export const ClickHouse: Plugin = {
+  name: "ClickHouse",
+  description:
+    "Grouparoo's ClickHouse integration allows you to sync all of your customer data that lives in your ClickHouse database with various destinations such as CRMs, Customer Support Tools, and Marketing Tools.",
+  priority: 80,
```

Comment:
This is interesting - I was wondering why Clickouse wasn't appearing as a source on the destination integration pages 
<img width="1367" alt="Screen Shot 2021-11-16 at 7 37 17 AM" src="https://user-images.githubusercontent.com/303226/142016173-b964712c-9943-4b72-93ce-362b0b653fd0.png">

But we only show the first 5 by priority.  https://github.com/grouparoo/www.grouparoo.com/blob/dd1f05b4576b4d79970e23d7c83d8df81e36fdbc/components/home/IntegrationSources.tsx#L17

This OK for now and we can re-shuffle the priorities later if we want

Reply:
Yep, I noticed that as well. I didn't want to prioritize clickhouse over the current ones being prioritized. I do think that should be a separate change